test/browser_test.dart

```diff
@@ -0,0 +1,47 @@
+@TestOn('browser')
+
+import 'package:js/js.dart';
+import 'package:node_interop/js.dart';
+import 'package:node_interop/util.dart';
+import 'package:sass/src/node/compile_options.dart';
+import 'package:test/test.dart';
+import 'ensure_npm_package.dart';
+import 'package:sass/src/node/compile_result.dart';
+
+@JS()
+external Sass get sass;
+
+@JS()
+class Sass {
+  external NodeCompileResult compileString(String text,
+      [CompileStringOptions? options]);
+  external Promise compileStringAsync(String text,
+      [CompileStringOptions? options]);
+}
+
+void main() {
+  setUpAll(ensureNpmPackage);
+  test('compileAsync() is not available', () {
+    expect(() => (sass as dynamic).compileAsync,
+        throwsA(isA<NoSuchMethodError>()));
+  });
+  test('compile() is not available', () {
+    expect(() => (sass as dynamic).compile, throwsA(isA<NoSuchMethodError>()));
+  });
+  test('render() is not available', () {
+    expect(() => (sass as dynamic).render, throwsA(isA<NoSuchMethodError>()));
+  });
+  test('renderSync() is not available', () {
+    expect(
+        () => (sass as dynamic).renderSync, throwsA(isA<NoSuchMethodError>()));
+  });
```

Comment:
It looks like these tests don't work. I wanted to assert that these methods are not included in the ESM export, but even when the export includes the methods these will pass. It looks like I'm just asserting the `Sass` type defined above.

@nex3 that makes me wonder: do we want to completely omit these methods from the ESM export, or include them but have them raise an error? For the former, how can I test for their absence here? And for the latter, what should the error say?

Reply:
I think throwing a runtime error is better. It should say something like "<method name> is only supported on Node.js."

Reply:
Done in 716f785c1891be3da06b3388cb318d0fbf2078ed and bbc867244c9c40a52f3210302891864cde0fc208